modules/storage-ephemeral-storage-overview.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// storage/understanding-persistent-storage.adoc[leveloffset=+1]
+
+[id=storage-ephemeral-storage-overview_{context}]
+= Overview
+
+:FeatureName: Ephemeral storage
```

Comment:
@RobertKrawitz, can you please check if this page still applies to OCP 4.x?

Reply:
Hi @RobertKrawitz and @rphillips - i just want to confirm that Ephemeral storage (Tech Preview) still applies to OCP 4?

Reply:
@RobertKrawitz BUMP - PTAL

Reply:
bump @sjenning as I am no longer working in this area.

Reply:
@sjenning or @mrunalp - bump to PTAL and let me know if Ephemeral storage (TP) still applies in OCP 4.x?

Reply:
It is not tech preview any more.  It is a GAed feature in upstream kube.

Reply:
Removed tech preview statement.